tfe_hardware/qcodes_instrument_drivers/Current_source/AD5760.py

```diff
@@ -0,0 +1,138 @@
+from functools import partial
+from typing import Optional, Union, Any
+import time,sys
+import clr  # install pythonnet
+
+from qcodes.instrument.parameter import DelegateParameter
+from qcodes import (Instrument, validators as vals)
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.instrument.channel import InstrumentChannel
+from qcodes.utils.validators import Numbers, Bool, Enum, Ints
+
+
+
+class AD5760(Instrument):
+
+    def __init__(self, name: str, host_name: str, path: str, syspath: str, terminator: str = "\n",
+                 **kwargs: Any) -> None:
+        super().__init__(name, **kwargs)
+        self.connect_board(host_name, path, syspath)
+
+        self.add_parameter('output',
+                           label='Output State',
+                           get_cmd=self.state,
+                           set_cmd=lambda x: self.on() if x else self.off(),
+                           val_mapping={
+                               'off': 0,
+                               'on': 1,
+                           })
+        
+        self.add_parameter('voltage',
+                           label='Voltage',
+                           unit='V',
+                           set_cmd=self._get_set_volt,
+                           get_cmd=self._get_volt,
+                           vals= Numbers(min_value=-10, max_value=10)
+                           )
+        # self.add_parameter('current',
+        #                    label="Current",
+        #                    unit='A',
+        #                    set_cmd=self._get_set_current,
+        #                    get_cmd=self._get_amp
+        # #                   )
+        # self._get_set_volt(5)
+        
+    def on(self) -> None:
+        self.write('OUTPUT 1')
+        self.measure._output = True
+        
+    def off(self) -> None:
+        self.write('OUTPUT 0')
+        self.measure._output = False
+
+    def state(self):
+        state = self.ask()
+        if state is not None:
+            return bool(1)
+        else:
+            return bool(0)
+    
+    def connect_board(self, host_name: str, path: str, syspath: str):
+        
+        sys.path.append(rf'{syspath}')
+        
+        # noinspection SpellCheckingInspection
+        
+        # noinspection SpellCheckingInspection
+        clr.AddReference('AnalogDevices.Csa.Remoting.Clients')
+        clr.AddReference('AnalogDevices.Csa.Remoting.Contracts')
+        
+        # noinspection PyUnresolvedReferences,SpellCheckingInspection
+        from AnalogDevices.Csa.Remoting.Clients import ClientManager  #noqa
```

Comment:
is there a reason to import here and not on the top with all the other imports?

Reply:
Yes, without adding the references, we cannot import client manager, lemme check whther I can add the references earlier.

Reply:
without setting the path, we cannot initialize the analog devices

Reply:
wait, I am talking about the import...
The comment is on the last line shown not the first, I don't know why github does it like that

Reply:
Yes, for the import you need to add the application as to the python system paths and then using clr we add this package to talk to the application. I don't really see a way around as this does not work if I Import it in the beginning